src/FSharp.Data.GraphQL.Server/Planning.fs

```diff
@@ -310,7 +305,7 @@ let private planVariables (schema: ISchema) (operation: OperationDefinition) =
                 { VarDef.Name = vname; TypeDef = idef; DefaultValue = vdef.DefaultValue }
             | _ -> raise (MalformedQueryException (sprintf "GraphQL query defined variable '$%s' of type '%s' which is not an input type definition" vname (tdef.ToString()))))
 
-let internal planOperation documentId (ctx: PlanningContext) (operation: OperationDefinition) : ExecutionPlan =
+let internal planOperation (ctx: PlanningContext) : ExecutionPlan =
```

Comment:
Here I refactored  PlanningContext to hold documentId and the operation definition as well. This makes this object be able to hold all needed information to plan an operation. Also, makes the middleware signature simpler.